plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/tool/display/ToolProxy.java

```diff
@@ -709,6 +714,14 @@ public void setSelectionProvider(IMapEditorSelectionProvider selectionProvider)
 		throw new UnsupportedOperationException("Call ToolProxy.getTool().setSelectionProvider(IMapEditorSelectionProvider) method instead"); //$NON-NLS-1$
 		
 	}
-    
-    
+
+    public boolean isToggleButton() {
```

Comment:
Let's add `@Override` for both, `isToggleButton()` and `isToggelingButtonEnabled()` as done for other methods from interface